roles/ovirt-cluster-upgrade/library/ovirt_scheduling_policies_facts.py

```diff
@@ -50,9 +50,9 @@
 # Examples don't contain auth parameter for simplicity,
 # look at ovirt_auth module to see how to reuse authentication:
 
-# Gather facts about all scheduling policies with name InClusterUpgrade:
+# Gather facts about all scheduling policies with name ClusterInMaintenance:
 - ovirt_scheduling_policies_facts:
-    name: InClusterUpgrade
+    name: ClusterInMaintenance
```

Comment:
in <4.1.4 it should not use/change any policy. As long as the code skips over those change/restore policy parts it's good enough